### PR TITLE
Add angle export option to UI

### PR DESCRIPTION
**Pull request by vpzomtrrfrt**

### CODE REVIEW

File: src/views/PopupView.tsx [hunk 1]
```diff
@@ -42,6 +44,8 @@ export default function PopupView(
                     <dd onMouseOver={()=>setControlsHelpImgNum(1)}>selection</dd>
                     <dt>Body Segments</dt>
                     <dd><input type={"checkbox"} checked={showSegments} onChange={()=>setShowSegments(!showSegments)} /></dd>
+                    <dt>Export</dt>
+                    <dd onClick={exportAngles}>angles</dd>
```
Comment (PTLux1517) [inline src/views/PopupView.tsx:48]: Could we make this a button to make it more obvious that it is a clickable element? Something to this effect:
```html
<dt>File Output</dt>
<dd><button onClick={exportAngles}>export angles</button></dd>
```

Comment (javitrejo14) [reply src/views/PopupView.tsx:48]: It looks like this change was added?

**Review by ConnorJacksonBSU — APPROVED** [review]

File: src/views/PopupView.tsx (hunk 1 above)
Comment (PTLux1517) [reply src/views/PopupView.tsx:48]: It hasn't yet. It's not currently a button

Comment (vpzomtrrfrt) [reply src/views/PopupView.tsx]: not sure what to think of this but it's a button now

**Review by PTLux1517 — APPROVED** [review]

**Pull request merged**